server/analytic.go

```diff
@@ -1,64 +0,0 @@
-package main
-
-import (
-	"sync"
-	"time"
-)
-
-type Analytic struct {
```

Comment:
There should be a GoDoc comment for this struct and it fields.

Reply:
:heavy_check_mark: